build.py

```diff
@@ -0,0 +1,143 @@
+#   -*- coding: utf-8 -*-
+import logging
+from pathlib import Path
+
+from pybuilder.core import use_plugin, init, before, after, Author
+from pybuilder.errors import BuildFailedException
+
+use_plugin("python.core")
+use_plugin("python.flake8")
+use_plugin("python.distutils")
+# use_plugin("python.unittest")
+# use_plugin("python.coverage")
+# https://github.com/AlexeySanko/pybuilder_pytest
+# use_plugin('pypi:pybuilder_pytest')
+# https://github.com/AlexeySanko/pybuilder_pytest_coverage
+# use_plugin('pypi:pybuilder_pytest_coverage')
+
+
+def _get_pysparkling_version() -> str:
+    """Returns the pysparkling version based on the one mentioned in the __init__.py file under the package."""
+    from pathlib import Path
+    import re
+
+    with open(Path(__file__).parent / 'src/pysparkling/__init__.py') as fp:
+        version_line = next(l for l in fp if l.startswith('__version__'))
+        return re.search(r'=\s*(["\'])(.*)(\1)\s*$', version_line).group(2)
```

Comment:
Why not importing the module? speed?
Would it make more sense to split the version definition to another file and import it both in here and in pysparkling/__init__.py?

Reply:
Exactly, loading a module loads everything under there. So opening the file is faster. Another way is to provide a __version__.py (which is often done) and move the __version__ in there.

Reply:
Although this should be part of the code clean imho.

Reply:
Indeed, I thought this code was introduced in this PR